client/src/components/Dashboard.js

```diff
@@ -13,10 +13,51 @@ export class Dashboard extends Component {
   }
 
   componentDidMount() {
+    this.setDocuments();
+  }
+
+  setDocuments() {
     getDocuments()
       .then(documents => this.setState({ documents }))
       .catch(err => console.log(err));
   }
+  handleErrors(response) {
+    return response.json().then(json => {
+      if (!response.ok) {
+        throw json.message;
+      } else {
+        return json;
+      }
+    });
+  }
+
+  pickDocument(id) {
+    const userRequest = {
+      method: "POST",
+      body: JSON.stringify({
+        document_id: id
+      }),
+      headers: {
+        Authorization: `Bearer ${sessionStorage.getItem("token")}`,
+        "Content-Type": "application/json"
+      }
+    };
+
+    fetch("http://localhost:4000/api/translations", userRequest)
```

Comment:
This will not work in Heroku, this url is okay just locally. Remove the localhost bit.

Reply:
it is done